client/src/components/DataFiles/DataFilesBreadcrumbs/DataFilesBreadcrumbs.jsx

```diff
@@ -155,52 +199,89 @@ const DataFilesBreadcrumbs = ({
       }
     }, '');
 
+  const lastTwoPaths = paths.slice(-1);
+  const lastTwoPathComps = pathComps.slice(-1);
+  const reversedPath = paths.reverse();
+
   return (
-    <div className={`breadcrumbs ${className}`}>
-      {scheme === 'projects' && (
-        <>
-          <RootProjectsLink
-            api={api}
-            section={section}
-            operation={operation}
-            label="Shared Workspaces"
-          />{' '}
-          {system && `/ `}
-        </>
+    <div className="breadcrumb-container">
+      <div id="path-button-wrapper">
+        <ButtonDropdown
+          isOpen={dropdownOpen}
+          toggle={toggleDropdown}
+          id="go-to-button-dropdown"
+          className="go-to-button-dropdown"
+        >
+          <DropdownToggle tag={Button}>Go to ...</DropdownToggle>
+          <DropdownMenu>
+            {reversedPath.slice(1, reversedPath.length).map((path, index) => {
+              const folderName = path.split('/').pop();
+              return (
+                <>
+                  <Link key={index} to={handleNavigation(path)}>
+                    <DropdownItem>
+                      <i className="icon-folder" />
+                      {folderName.length > 20
+                        ? folderName.substring(0, 20)
+                        : folderName}
+                    </DropdownItem>
+                  </Link>
+                </>
+              );
+            })}
+            <DropdownItem divider />
+            <Link to={handleNavigation(homeDir)}>
+              <DropdownItem>
+                <i className="icon-my-data" />
+                <span className="multiline-menu-item-wrapper">
+                  {systemName || 'Shared Workspaces'}
+                  <small>Root</small>
+                </span>
+              </DropdownItem>
+            </Link>
+          </DropdownMenu>
+        </ButtonDropdown>
+      </div>
+      <div className={`breadcrumbs ${className}`}>
+        {lastTwoPathComps.length === 0 ? (
+          <span className="system-name truncate">
+            {systemName || 'Shared Workspaces'}
+          </span>
+        ) : (
+          lastTwoPathComps.map((pathComp, i) => {
+            if (i === lastTwoPaths.length - 1) {
+              return (
+                <span key={uuidv4()} className="truncate">
+                  {pathComp}
+                </span>
+              );
+            }
```

Comment:
Should this be showing the last two paths? I'm only seeing the current directory
![Screenshot 2023-10-04 at 7 10 03 PM](https://github.com/TACC/Core-Portal/assets/20326896/5351605f-28b5-4389-9005-30cc92faf9dc)

Reply:
Possible relevant are [designer comments that mentions the two path thing (internal link)](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1695227961355839?thread_ts=1695129819.592429&cid=CQUA4D5KJ).

Reply:
Ah makes sense! Yes renaming to something more accurate to it's function sounds good 👍